ide/static/js/content.js

```diff
@@ -122,8 +143,16 @@ class Content extends React.Component {
 
   modifyLayer(layer, layerId = this.state.selectedLayer) {
     const net = this.state.net;
+    var oldLayerParams = this.state.totalParameters;
```

Comment:
Instead of repeating this portion of the code from `modifyLayerParams` can we have a separate function which has this in one place and both these functions can call it.

Reply:
I modified old logic where we have to update layer after param pane is closed which does exactly the same thing you are asking for, now we don't call modifyLayer from modifyLayerParams